src/main/scala/mesosphere/marathon/core/task/tracker/impl/AppData.scala

```diff
@@ -32,9 +33,8 @@ private[impl] case class AppDataMap private (appTasks: Map[PathId, AppData]) {
     task <- app.taskMap.get(taskId)
   } yield task
 
-  def toTaskTrackerAppMap: Map[PathId, TaskTracker.App] = {
-    appTasks.mapValues(_.toTaskTrackerApp)
-  }
+  def toTaskTrackerAppMap: Map[PathId, TaskTracker.App] = appTasks.mapValues(_.toTaskTrackerApp)
+  def tasks: Iterable[MarathonTask] = appTasks.view.flatMap(_._2.tasks)
```

Comment:
nit-picking: in other locations, we have names like `allIds` (which I'm not necessarily fond of). Here, we have `getTasks(appId)` vs `tasks`. What do you think about aligning and calling it `allTasks` instead? Otherwise, I'd tend to use `tasksFor(appId)` and `tasks` to make it explicit.